mkdocs/docs/HPC/rlibs.md

```diff
@@ -0,0 +1,40 @@
+# R libraries
+
+Please keep in mind, that this is not a general manual
+abut installing `R` libraries, but only system specific information about self 
+installed R libraries at {{ hpcinfra }}.
+
+Our infrastructure contains different CPU architectures, that are
+not necessarily compatible with each other. Additionally, we have 
+multiple versions of `R` installed with different versions of toolchains, and by default,
+self installed libraries are installed in separate directories based only
+on `R` major and minor versions.
+
+In this way, you might use self installed `R` libraries on incompatible architectures and/or
+compiled with incompatible toolchains, which might lead to hangs or other types of errors.
+
+When you load a centrally installed `R` module we make sure that self installed libraries 
+will be installed to an operational system, architecture, toolchain, and `R` version
+dependent directory. The location of this directory by default is
```

Comment:
```suggestion
Since end of February 2025, we make sure that `R` packages that you install yourself will be installed
in a directory that is specific to the operating system version, CPU microarchitecture, toolchain,
and `R` version that is being used. The location of this directory by default is